saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/nlp/EntityRelation/EntityRelationSensors.scala

```diff
@@ -20,16 +20,16 @@ object EntityRelationSensors {
   lazy val personGazetSensor = new GazeteerReader("gazeteer/known_maleFirst.lst", "Gaz:Person", true, true)
   personGazetSensor.addFile("gazeteer/known_femaleFirst.lst", true, true)
 
-  def readConllData(dir: String): (List[ConllRawSentence], List[ConllRelation], List[ConllRawToken]) = {
-    val reader = new Conll04_Reader(dir, "Token")
+  def readConllData(dir: String, sentenceIdOffset: Int = 0): (List[ConllRawSentence], List[ConllRelation], List[ConllRawToken]) = {
+    val reader = new Conll04_Reader(dir, "Token", sentenceIdOffset)
```

Comment:
Interfacing the data structures of LBJava and SL makes a mess for the feature extraction part. Does anyone dare to check with me and help me debugging this?